mitiq/calibration/calibrator.py

```diff
@@ -236,7 +264,7 @@ def convert_to_expval_executor(executor: Executor, bitstring: str) -> Executor:
         the most likely bitstring, according to the passed ``distribution``
     """
 
-    def expval_executor(circuit: cirq.Circuit) -> float:
+    def expval_executor(circuit: QPROGRAM) -> float:
```

Comment:
After this change, is the type ignore comment still needed in the return statement of this function?